client/src/components/employer-form/index.js

```diff
@@ -13,7 +14,8 @@ import { Summary } from './Summary';
 import { OperatorInfo } from './OperatorInfo';
 import { SiteInfo } from './SiteInfo';
 import { HcapRequest } from './HcapRequest';
-import { BusinessDetails } from './BusinessDetails';
+
```

Comment:
so minor, but I don't think we want this newline :) the other form components are all grouped above